coalib/parsing/DefaultArgParser.py

```diff
@@ -182,18 +182,25 @@ def default_arg_parser(formatter_class=None):
                                    action='store_const',
                                    help=SHOW_BEARS_HELP)
 
-        outputs_group.add_argument('-A',
-                                   '--show-all-bears',
-                                   const=True,
-                                   action='store_const',
-                                   help="Display all bears.")
-
-        SHOW_LANGUAGE_BEARS = ("Display all bears for the given languages.")
         outputs_group.add_argument('-l',
-                                   '--show-language-bears',
+                                   '--filter-by-language',
                                    nargs='+',
                                    metavar='LANG',
-                                   help=SHOW_LANGUAGE_BEARS)
+                                   help="Filters the output of `--show-bears` "
+                                        "by the given languages.")
```

Comment:
Marked https://github.com/coala-analyzer/coala/issues/2311 as low instead of newcomer since I've also asked for parsing comma-separated language input.